packages/now-static-build/src/index.ts

```diff
@@ -407,10 +415,35 @@ export async function build({
   throw new Error(message);
 }
 
-export async function prepareCache({ workPath }: PrepareCacheOptions) {
-  return {
-    ...(await glob('node_modules/**', workPath)),
-    ...(await glob('package-lock.json', workPath)),
-    ...(await glob('yarn.lock', workPath)),
-  };
+export async function prepareCache({
+  entrypoint,
+  workPath,
+}: PrepareCacheOptions) {
+  let cachePaths: string[] = [
+    'node_modules/**',
+    'package-lock.json',
+    'yarn.lock',
+  ];
+
+  // add framework specific cache paths
+  const pkg = getPkg(entrypoint, workPath);
+  if (pkg) {
+    const framework = getFramework(pkg);
+
+    if (framework && framework.cachePaths) {
+      cachePaths = [...cachePaths, ...framework.cachePaths];
+    }
+  }
+
+  // glob all cache paths and return
+  let cacheFiles = {};
+
+  for (const cachePath of cachePaths) {
+    cacheFiles = {
+      ...cacheFiles,
+      ...(await glob(cachePath, workPath)),
+    };
```

Comment:
I think `Object.assign(cacheFiles, await glob(cachePath, workPath));` would be more straight forward here 🤔